packages/cloudfront-security-headers/README.md

```diff
@@ -1,8 +1,18 @@
 # AWS CDK CloudFront Security Headers
-This package contains a Lambda@Edge function for cloudfront to add security headers to the origin response of all requests.
+![TypeScript version](https://img.shields.io/github/package-json/dependency-version/aligent/cdk-constructs/dev/typescript?filename=packages/cloudfront-security-headers/package.json&color=red) ![AWS CDK version](https://img.shields.io/github/package-json/dependency-version/aligent/cdk-constructs/dev/aws-cdk?filename=packages/cloudfront-security-headers/package.json) ![NPM version](https://img.shields.io/npm/v/%40aligent%2Fcdk-cloudfront-security-headers?color=green)
 
-## Useful commands
+This package contains a Lambda@Edge function for Cloudfront to add security headers to the origin response of all requests.
 
- * `npm run build`   compile typescript to js
- * `npm run watch`   watch for changes and compile
- * `npm run test`    perform the jest unit tests
\ No newline at end of file
+The function is intended to be added to an existing Cloudfront. 
+
+## Usage and Default Options
+### `contentSecurityPolicy` (array&lt;string&gt;, optional)
+- Array to store content security policies to attach
+
+## Security headers attached
+```
+headers["strict-transport-security"]
```

Comment:
It's important to know what the values of these are. E.g. that `x-content-type-options` is set to `nosniff`